### DISCUSSION TITLE
Server-side DataTable for big data

### DISCUSSION THREAD

**Discussion by LukaTarkhn** [primefaces/community#763] [PrimeVue] [answered]
Why PrimeVue does not have this variant of the data table, that meant to be used for very large datasets, where it would be inefficient to load all the data into the client. It supports sorting, filtering, pagination, and selection like a standard data table, but all the logic must be handled externally by your backend or database?

**Comment by tugcekucukoglu** [comment, marked as answer]
[Lazy loading](https://primevue.org/datatable/#lazy_load) can be helped for better performance.

**Reply by LukaTarkhn** [reply]
Lazy loading works only for api or with iniertia to?
